### DISCUSSION TITLE
Installing this on my new laravel project.

### DISCUSSION THREAD

**Discussion by codelyftlab** [savannabits/filament-modules#54] [Q&A]
Good evening, Im a bit lost at the documentation. It is said that i need to install the nwidart and filament first right ? And I did. I also install my filament panel with id "Dashboard". 

Here is the part that onfuse me 

```
#Generate a new Panel inside your module
php artisan module:make-filament-panel admin Blog # php artisan module:make-filament-panel [id] [module]
```

Does that means I must install a panel inside every modules ? What happend if I have more than 1 modules ?

**Comment by coolsam726** [comment]
This has changed since. Please check the documentation for v4 and v5 for an update.
